### PR TITLE
DNANexus docs

### PR DESCRIPTION
**Pull request by LiterallyUniqueLogin**

### CODE REVIEW

File: doc/DNANexus.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+DNANexus & UK Biobank Research Analysis Platform (RAP)
+======================================================
+
+Last update: 2023/03/16
+
+Getting access to the RAP
+-------------------------
+
+#. Sign up `here <https://bbams.ndph.ox.ac.uk/ams/resProjects>`_
```
Comment (aryarm) [inline doc/DNANexus.rst:9]: these bullets might show up a bit strangely?
image

File: doc/DNANexus.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+------------------------------------
+(There's also a GUI, docs TODO)
+
+1. Install the DNA nexus command line tools vended through pip: :code:`pip3 install dxpy`.
+2. Run :code:`dx login` and :code:`dx select <project name>`.
+
+Choosing Instance Types
+-----------------------
+Instances are the virtual machines allocated for each job in the cloud.
+The link `here <https://dnanexus.gitbook.io/uk-biobank-rap/working-on-the-research-analysis-platform/billing-and-costs#rates>`_
```
Comment (aryarm) [inline doc/DNANexus.rst:27]: Try this to resolve the build failures
```suggestion
The link `here <https://dnanexus.gitbook.io/uk-biobank-rap/working-on-the-research-analysis-platform/billing-and-costs#rates>`__
```

File: doc/DNANexus.rst (hunk 1 above)
Comment (aryarm) [inline doc/DNANexus.rst:9]: Try this to resolve the build failures
```suggestion
#. Sign up `here <https://bbams.ndph.ox.ac.uk/ams/resProjects>`__
```

**Review by aryarm — CHANGES_REQUESTED** [review]
there were a few warnings that came up when the system tried to build the docs
I think these suggestions I made should fix them

also, I think you need to add the new `DNANexus.rst` to the table of contents in `index.rst`?

File: doc/DNANexus.rst (hunk 1 above)
Comment (aryarm) [reply doc/DNANexus.rst:9]: ah, I just meant that the numbers are all 1

Should they be 1, 2, 3, etc?

**Review by aryarm — APPROVED** [review]

**Comment by aryarm** [comment]
looks like it's good to go!

**Review by yli091230 — APPROVED** [review]

**Comment by yli091230** [comment]
@aryarm Thanks for helping with the review!

**Pull request merged**